media-proxy/include/mesh/st2110tx.h

```diff
@@ -0,0 +1,153 @@
+#ifndef ST2110TX_H
+#define ST2110TX_H
+
+#include "st2110.h"
+#include <algorithm>
+
+namespace mesh::connection {
+
+/**
+ * ST2110
+ *
+ * Base abstract class of ST2110. ST2110_20Tx/ST2110_22Tx/ST2110_30Tx
+ * inherit this class.
+ */
```

Comment:
The comment is wrong. It has been copied from the base class.

Reply:
Fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/261/commits/d6fb54f34d3b43543423c71401f5fca75f1fbdde